### DISCUSSION TITLE
https://klp1.com/issues/loose-belt/

### DISCUSSION THREAD

**Discussion by giscus** [blakadder/klp1#2] [Announcements]
# https://klp1.com/issues/loose-belt/

How to tighten the loose belts on a Kingroon KLP1

https://klp1.com/issues/loose-belt/

**Comment by icreated4333** [comment]
Hi there, this is excellent! Thank you for all of the data you have provided here. I run the Kingroon discord server and would really appreciate if you could share the issues and possible fixes there. 
-Icreated

**Reply by blakadder** [reply]
and that server is.........

**Reply by icreated4333** [reply]
Hi, apologies for not actually posting a link, here it is https://discord.gg/ryeTUUuXmN
